terraform/projects/app-ecs-services/task-definitions/git-puller.json

```diff
@@ -11,6 +11,14 @@
         "containerPath": "/configs"
       }
     ],
-    "command": ["clone", "https://github.com/alphagov/prometheus-config.git", "/configs"]
+    "command": ["clone", "https://github.com/alphagov/prometheus-config.git", "/configs"],
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "${log_group}",
+        "awslogs-region": "${region}",
+        "awslogs-stream-prefix": "git-puller"
```

Comment:
Do we need the stream prefixes? They seem kind of redundant as it is a duplicated in the log stream name

Reply:
I tried without and it turns out that specifying a stream prefix has nontrivial extra behaviour which we want. see 148fc06 for details.

Reply:
Thanks for investigating 👍